generate_labels.py

```diff
@@ -0,0 +1,25 @@
+import csv
+from os import listdir
+
+from utils import read_labels
+
+
+labels_directory = "label_persons/"
+all_file_names = [f for f in listdir(labels_directory)]
+
+labels_voted = {}
+ydn_map = {'y': 0, 'd': 1, 'n': 2}
+ydn_reverse_map = {0: 'y', 1: 'd', 2: 'n'}
+
+for file in all_file_names:
```

Comment:
Can you use a different variable name here? `file` is a Python keyword so it gets highlighted differently than other variable (which can be confusing). Maybe use `file_name`